roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -32,10 +32,10 @@
         ) |
         combine(
           {
-            'DATAPLANE_REPO': operators_build_output['dataplane-operator'].git_src_dir,
-            'DATAPLANE_BRANCH': '',
+            'OPENSTACK_REPO': operators_build_output['openstack-operator'].git_src_dir,
+            'OPENSTACK_BRANCH': '',
             'GIT_CLONE_OPTS': '-l',
-          } if ('dataplane-operator' in operators_build_output)
+          } if ('openstack-operator' in operators_build_output)
```

Comment:
probably we just need to get rid of the combine since we have OPENSTACK_REPO and OPENSTACK_BRANCH already defined.